src/index.html

```diff
@@ -1,16 +1,53 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="attr-req-value" 
+          method="post" 
+          class="form"
+          data-qa="big"
+    >
```

Comment:
```suggestion
    <form 
      action="attr-req-value" 
      method="post" 
      class="form"
      data-qa="big"
    >
```
вивправ форматування всюди